src/translations/en.json

```diff
@@ -2332,6 +2332,13 @@
             },
             "provided_by_custom_integration": "Provided by a custom integration",
             "depends_on_cloud": "Depends on the cloud",
+            "quality_scale": "Ranked {quality_scale} on our quality scale",
```

Comment:
Perhaps use `quality_scale_note` (or similar) as the translation key so that `quality_scale` remains free in case we want to translate the term "Quality scale" in the UI at some point?

Reply:
If that is the case, I would expect that key to be under `integrations` not under `integrations.config_entry` where this is